### PR TITLE
Improved template for playground

### PR DESCRIPTION
**Pull request by mmarcon**
## Description
Improved template to set the right expectations on what one can do with playgrounds.

image

### Checklist
- [ ] New tests and/or benchmarks are included
- [x] Documentation is changed or added

## Motivation and Context
We want to onboard users to playgrounds by showing them what kind of operations and APIs are supported.

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [x] Misc

## Types of changes

- [ ] *Backport Needed*
- [x] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/templates/playgroundTemplate.ts [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+  { "_id" : 6, "item" : "def", "price" : 7.5, "quantity": 5, "date" : new Date("2015-06-04T05:08:13Z") },
+  { "_id" : 7, "item" : "def", "price" : 7.5, "quantity": 10, "date" : new Date("2015-09-10T08:43:00Z") },
+  { "_id" : 8, "item" : "abc", "price" : 10, "quantity" : 5, "date" : new Date("2016-02-06T20:20:13Z") },
+]);
 
 // Run a find command.
-db.myCollection.find({ foo: 'bar' });
+db.sales.find({ date: { $gte: new Date("2014-04-04"), $lt: new Date("2014-04-05") } });
 
 // Run an aggregation.
```
Comment (fredtruman) [inline src/templates/playgroundTemplate.ts]: Should we describe what the aggregation is doing?
// Run an aggregation to view total sales for each product in 2014

File: src/templates/playgroundTemplate.ts [hunk 2]
```diff
@@ -1,15 +1,30 @@
-const template: string = `// Select the database to use.
-use('test');
+const template: string = `// MongoDB Playground
+// To disable this template go to Settings | MongoDB | Use Default Template For Playground.
+
+// Select the database to use.
+use("test");
+
+// Insert a few documents.
```
Comment (fredtruman) [inline src/templates/playgroundTemplate.ts]: // Insert a few documents into the _sales_ collection.

File: src/templates/playgroundTemplate.ts [hunk 3]
```diff
@@ -1,15 +1,30 @@
...
+  { "_id" : 2, "item" : "jkl", "price" : 20, "quantity" : 1, "date" : new Date("2014-03-01T09:00:00Z") },
+  { "_id" : 3, "item" : "xyz", "price" : 5, "quantity" : 10, "date" : new Date("2014-03-15T09:00:00Z") },
+  { "_id" : 4, "item" : "xyz", "price" : 5, "quantity" :  20, "date" : new Date("2014-04-04T11:21:39.736Z") },
+  { "_id" : 5, "item" : "abc", "price" : 10, "quantity" : 10, "date" : new Date("2014-04-04T21:23:13.331Z") },
+  { "_id" : 6, "item" : "def", "price" : 7.5, "quantity": 5, "date" : new Date("2015-06-04T05:08:13Z") },
+  { "_id" : 7, "item" : "def", "price" : 7.5, "quantity": 10, "date" : new Date("2015-09-10T08:43:00Z") },
+  { "_id" : 8, "item" : "abc", "price" : 10, "quantity" : 5, "date" : new Date("2016-02-06T20:20:13Z") },
+]);
 
 // Run a find command.
```
Comment (fredtruman) [inline src/templates/playgroundTemplate.ts]: // Run a find command to view items sold on April 4th, 2014.

**Comment by fredtruman** [comment]
Nice Max, looks great. Just a couple copy/comment suggestions ...

**Comment by mmarcon** [comment]
Made the changes. I feel like in the future we may want to make this template a bit more _essential_ so users can use it as a starting point without having to delete to much (instead of using the empty template) but for the time being I like that it contains a very concrete use case and has everything in there.

File: src/templates/playgroundTemplate.ts [hunk 4]
```diff
@@ -16,10 +16,10 @@ db.sales.insertMany([
   { "_id" : 8, "item" : "abc", "price" : 10, "quantity" : 5, "date" : new Date("2016-02-06T20:20:13Z") },
 ]);
 
-// Run a find command.
+// Run a find command to view items sold on April 4th, 2014.
 db.sales.find({ date: { $gte: new Date("2014-04-04"), $lt: new Date("2014-04-05") } });
 
-// Run an aggregation.
+// to view total sales for each product in 2014.
```
Comment (fredtruman) [inline src/templates/playgroundTemplate.ts]: @mmarcon the "Run an aggregation ..." got dropped here?

Comment (mmarcon) [reply src/templates/playgroundTemplate.ts]: Oh, not intentional. Nice catch!

**Comment by mmarcon** [comment]
@Anemy @alenakhineika @imlucas if this LGTY once the checks have passed we can merge.

**Pull request merged**